Sources/active-win/main.swift

```diff
@@ -5,9 +5,21 @@ func toJson<T>(_ data: T) throws -> String {
 	return String(data: json, encoding: .utf8)!
 }
 
+if !AXIsProcessTrustedWithOptions(["AXTrustedCheckOptionPrompt": true] as CFDictionary) {
+    print("BActive-Win requires permission for accessibility in System Preferences > Security & Privacy > Privacy > Accessibility.")
+    exit(0)
+}
+
 let frontmostAppPID = NSWorkspace.shared.frontmostApplication!.processIdentifier
 let windows = CGWindowListCopyWindowInfo([.optionOnScreenOnly, .excludeDesktopElements], kCGNullWindowID) as! [[String: Any]]
 
+let windowImage = CGWindowListCreateImage(.null, .optionIncludingWindow, windows[0][kCGWindowNumber as String] as! CGWindowID, [.boundsIgnoreFraming, .bestResolution])
```

Comment:
It should be clearer from the naming here that this is to trigger the permission dialog.